css/reveal.scss

```diff
@@ -1073,7 +1073,7 @@ $controlsArrowAngleActive: 36deg;
 	background: black;
 	visibility: hidden;
 	opacity: 0;
-	z-index: 100;
+	z-index: 10001;
```

Comment:
Instead of bumping z-index up higher, can we leave it at 100 and move the overlay down to 99? Thanks!

Reply:
Yes totally, I had that before. The reason I switch to moving the pause overlay up, was that I assume the overlay function (begin generic) has potentially being customised by users at more places, and therefore a change to the z-index could harm quit a bit. At the same time, when user have overridden the `z-index` of the pause overlay already, this change will not affect them. 

Either way, happy to follow your recommendation.

Reply:
It's a good consideration but let's go ahead and change that anyway. It's such a small change I doubt it will affect many people.